app/src/main/java/com/bunbeauty/papakarlo/feature/consumercart/ui/CountPicker.kt

```diff
@@ -47,12 +55,32 @@ fun CountPicker(
             descriptionStringId = R.string.description_consumer_cart_decrease,
             onClick = onCountDecreased
         )
-        Text(
-            modifier = Modifier.padding(horizontal = 4.dp),
-            text = count.toString(),
-            style = FoodDeliveryTheme.typography.bodySmall.bold,
-            color = FoodDeliveryTheme.colors.mainColors.primary
-        )
+
+        AnimatedContent(
```

Comment:
- Set `.height(IntrinsicSize.Min)` for container
- Remove `.height(36.dp)` for `AnimatedContent`
- Remove `contentAlignment = Alignment.Center` for `AnimatedContent`